src/Microsoft.ML.Data/DataLoadSave/TransformWrapper.cs

```diff
@@ -123,9 +121,9 @@ private static bool IsChainRowToRowMapper(IDataView view)
             return true;
         }
 
-        public bool IsRowToRowMapper { get; }
+        bool ITransformer.IsRowToRowMapper => IsChainRowToRowMapper(_xf);
```

Comment:
>IsChainRowToRowMapper [](start = 46, length = 21)

This operation is somewhat involved, so I would prefer if you could have a backing field to store this result as we had in the initial implementation.

Reply:
Sounds good let me do that!

---
In reply to: [258775778](https://github.com/dotnet/machinelearning/pull/2673#discussion_r258775778) [](ancestors = 258775778)